internal/pkg/gitdirty/gitdirty.go

```diff
@@ -188,65 +212,78 @@ func remoteConvertHTTPStoSSH(httpsRemote string) (string, error) {
 // Based on regex, and may not match every possible style of remote, but tested on github and gitlab.
 //    Example input: git@git.test:testorg/testrepo.git
 //           output: https://git.test/testorg/testrepo.git
+//    Example input: git.test:testorg/testrepo.git
+//           output: https://git.test/testorg/testrepo.git
 func remoteConvertSSHtoHTTPS(sshRemote string) (string, error) {
 	if !isSSHRemote(sshRemote) {
 		return "", fmt.Errorf("%s is not an ssh remote", sshRemote)
 	}
 
-	httpsRemote := githubStyleSshRemoteRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	var httpsRemote string
+	// Check if it has git@
+	if githubStyleSshRemoteRegexp.MatchString(sshRemote) {
+		httpsRemote = githubStyleSshRemoteRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	} else {
+		// Doesn't have the git@ at the front of the url
+		httpsRemote = githubStyleSshRemoteNoAtRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	}
+
```

Comment:
Style nit: if it's going to have a "default" that's only changed in a certain condition, this format could make that simpler/clearer:

```suggestion
	httpsRemote := githubStyleSshRemoteNoAtRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
	// Check if it has git@
	if githubStyleSshRemoteRegexp.MatchString(sshRemote) {
		httpsRemote = githubStyleSshRemoteRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
	}
```

The current `if/else` is fine though so take it or leave it

Reply:
I left this explicit in the if/else bc then it acts in an incorrect manner by having a default val w/o the if/else. For ex: in the case where we have the `sshRemote = git@git.test:testorg/testrepo.git`, then by having a default it would make the `httpsRemote = https://git@git.test/testorg/testrepo.git` bc the regex is for `githubStyleSshRemoteNoAtRegexp` in the default case and the git@ is not accounted for properly in that regex. Then it would overwrite the value in the `if` correctly, but I didn't like that it would have the wrong value in the first place. Maybe there is another edge case that I didn't account for that could return the incorrect `httpsRemote` in the future. I wrote out the explicit if/else bc I couldn't in the `githubStyleSshRemoteNoAtRegexp` regex put ^(?!git@) which would help to rectify the above mentioned case in that we want to specifically use the `githubStyleSshRemoteRegexp` in cases where we have the git@ and use the `githubStyleSshRemoteNoAtRegexp` in cases where it is still of type ssh, but w/o the git@ explicitly.

Reply:
Ah yeah that makes sense 👍